fuzzers/utils.py

```diff
@@ -55,31 +74,23 @@ def append_flags(env_var, additional_flags, env=None):
     and assign env_var to the result."""
     if env is None:
         env = os.environ
-    flags = env.get(env_var, '').split(' ')
+
+    env_var_value = env.get(env_var)
+    flags = env_var_value.split(' ') if env_var_value else []
     flags.extend(additional_flags)
     env[env_var] = ' '.join(flags)
 
 
-# Use these flags when compiling benchmark code without a sanitizer (e.g. when
-# using eclipser). This is necessary because many OSS-Fuzz targets cannot
-# otherwise be compiled without a sanitizer because they implicitly depend on
-# libraries linked into the sanitizer runtime. These flags link against those
-# libraries.
-NO_SANITIZER_COMPAT_CFLAGS = [
-    '-pthread', '-Wl,--no-as-needed', '-Wl,-ldl', '-Wl,-lm',
-    '-Wno-unused-command-line-argument'
-]
-NO_SANITIZER_COMPAT_CXXFLAGS = ['-stdlib=libc++'] + NO_SANITIZER_COMPAT_CFLAGS
-
-
-def set_no_sanitizer_compilation_flags(env=None):
-    """Set compilation flags (CFLAGS and CXXFLAGS) in |env| so that a benchmark
-    can be compiled without a sanitizer. If |env| is not provided, the program's
-    environment will be used."""
-    if env is None:
-        env = os.environ
-    env['CFLAGS'] = ' '.join(NO_SANITIZER_COMPAT_CFLAGS)
-    env['CXXFLAGS'] = ' '.join(NO_SANITIZER_COMPAT_CXXFLAGS)
+def get_config_value(attribute):
+    """Gets config attribute value from benchmark config yaml file."""
+    with open(BENCHMARK_CONFIG_YAML_PATH) as file_handle:
+        # Hacky yaml parsing without PyYAML dependency.
```

Comment:
Why are we doing this? Why can't we use pyyaml. I think we should never ever do this.

Reply:
The reason being at fuzzer runtime, we use runner.Dockerfile from fuzzer directory and we can't install pyyaml dependency there.

Reply:
Are you talking about runtime or buildtime (double checking).
I didn't fuzzers/utils.py used to only be a buildtime thing.
In any case, I'm pretty sure we have pyyaml at runtime because fuzzer.py at runtime is invoked by experiment/runner.py which I believe has these dependencies.

Reply:
Adding at buildtime is okish, i can add requirements.txt for benchmark-builder.
but runtime, it was not working, let me check more.

Reply:
Fixed as per offline discussion.